eth/stagedsync/stage_polygon_sync.go

```diff
@@ -226,6 +226,14 @@ func UnwindPolygonSyncStage(ctx context.Context, tx kv.RwTx, u *UnwindState, cfg
 		return err
 	}
 
+	if err := stages.SaveStageProgress(tx, stages.Headers, u.UnwindPoint+1); err != nil {
```

Comment:
u.UnwindPoint+1 here does not match with `u.Done(tx)` which does `SaveStageProgress(u.UnwindPoint)` - is this a discrepancy or im misunderstanding? Same applies for Bodies unwind below